src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/comments/ProblematicYamlTest.kt

```diff
@@ -0,0 +1,168 @@
+package it.krzeminski.snakeyaml.engine.kmp.comments
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import it.krzeminski.snakeyaml.engine.kmp.api.LoadSettings
+import it.krzeminski.snakeyaml.engine.kmp.events.CommentEvent
+import it.krzeminski.snakeyaml.engine.kmp.events.Event
+import it.krzeminski.snakeyaml.engine.kmp.parser.Parser
+import it.krzeminski.snakeyaml.engine.kmp.parser.ParserImpl
+import it.krzeminski.snakeyaml.engine.kmp.scanner.StreamReader
+
+class ProblematicYamlTest : FunSpec({
+    val LOAD_OPTIONS: LoadSettings = LoadSettings.builder().setParseComments(true).build()
```

Comment:
Since it is a local variable, we should rename it accordingly

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/499/commits/51d227c180e068aa379c96b8057f0c227706d044